docs/backend_documentation/profile.md

```diff
@@ -0,0 +1,27 @@
+# Profile
+
```

Comment:
Could you add a short description of what the profile page will show? Basically what we talked about with the person's name, major, school, class (ie. 2027), and past opportunities.